vagrant/README.md

```diff
@@ -23,6 +23,22 @@ You can set up the Vagrant environment with just one command:
 ```bash
 vagrant up
 ```
+If you will experience following error 
```

Comment:
This is only relevant to MacOS, and I think we should point this out.

Reply:
Done. Check if that's correct convention